tensorboard/plugins/projector/polymer3/vz_projector/vz-projector-bookmark-panel.html.ts

```diff
@@ -0,0 +1,211 @@
+/* Copyright 2016 The TensorFlow Authors. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+==============================================================================*/
+
+import {html} from '@polymer/polymer';
+
```

Comment:
imports for:
iron-collapse/iron-collapse
paper-icon-button/paper-icon-button
paper-input/paper-textarea
paper-tooltip/paper-tooltip
?